src/lib/cron-builder.css

```diff
@@ -125,3 +125,6 @@
 .cron_builder {
     font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
 }
+.cursor_pointer {
```

Comment:
Can we move this inside `.cron_builder` class? 
I don't want this to be affected in parent project classes.

Reply:
I havnt completed my review/test. 
Please wait for me to complete my tests, so that you can fix all changes at once 😄.

Reply:
The pointer is not working for me 🤔. (using Mac M1, Chrome)
I will test same again in this evening!. 
 
https://github.com/user-attachments/assets/af4b730e-dd5d-4f57-a6e8-0137dcbdfd60

Reply:
Hey @sojinantony01 Let me know if there's the pointer issue; I tried it in multiple browsers, and it's working well for me. I'm using M2 & Chrome/Safari.
> Can we move this inside .cron_builder class?
> I don't want this to be affected in parent project classes.
Sure, I can add the same once testing/reviewing is finalised.

Reply:
Hi, Sorry for the delay. I have finished my tests
The mouse pointer is working after a restart!
Not sure why it was not working before 😄. 

I am ready to approve the PR once the class pointer is moved inside cron_builder
@shivam178

Reply:
Done!